src/app/Components/login/login.component.scss

```diff
@@ -0,0 +1,12 @@
+.login-form {
+    min-width: 150px;
+    max-width: 500px;
+    width: 100%; 
+  }
+  
+  .login-full-width {
```

Comment:
you should promote this at app level, in `style.css` so it can be shared and remove it from here